Formula/libomxil-bellagio.rb

```diff
@@ -26,10 +26,10 @@ def install
 
     # Be explicit about the configure flags
     args << "--enable-static=#{build.with?("static") ? "yes" : "no"}"
-
-    system "autoreconf", "-fiv"
+    
     system "./configure", *args
     ENV.deparallelize
+    ENV.delete "CFLAGS"
```

Comment:
Let's move this to the top of `def install`

Reply:
Ok.